posthog/api/utils.py

```diff
@@ -364,11 +368,7 @@ def select_event_definitions(fields: str):
 
     # Only return event definitions
     raw_event_definition_fields = ",".join(event_definition_fields)
-    ordering = (
-        "ORDER BY last_seen_at DESC NULLS LAST, query_usage_30_day DESC NULLS LAST, name ASC"
-        if is_enterprise
-        else "ORDER BY name ASC"
-    )
+    ordering = f"ORDER BY {order_UNSAFE} {order_direction} NULLS LAST, name ASC"
```

Comment:
Am torn on whether we should force nulls last.

But, not blocking. Some server sorting is better than none